### ISSUE TITLE
Gantt Progress Field Visual Bugs

### ISSUE THREAD

**Issue by nicola-dragoni** [microsoft/powerbi-visuals-gantt#180]
Hello,

One of my clients is experiencing a weird bug regarding the progress field in the gantt Visual.
The bar filling changes with regard to the filter context applied to the visual.

Taking a look at the screenshots provided, you can see that the highlighted row sometimes shows a fillin in the bar, and sometimes it does not.

The expected behaviour, in this case, would be to show no filling (progress value is zero)

Is there something that I am missing?
![Gantt_Bug2](https://user-images.githubusercontent.com/52753209/66809591-f1c07700-ef2d-11e9-816d-023f3b3d03fc.png)
![Gantt_Bug1](https://user-images.githubusercontent.com/52753209/66809593-f2590d80-ef2d-11e9-98e0-30184190c38c.png)

**Comment by helen508light** [comment]
Hello @nicola-dragoni
Could you provide the report reproducing this issue?
Does this issue exist in previous versions?

**Comment by nicola-dragoni** [comment]
Hello @helen508light;
Thank you for your reply.

I don't know wheter this issue existes with previous versions, my client raised this problem to me just now.


[Report.zip](https://github.com/microsoft/powerbi-visuals-gantt/files/3728288/Report.zip)

**Comment by kenneth817** [comment]
@helen508light ,please can you confirm when this issue can be resolved as I am facing a similar issue on the desktop version.Thanks

**Comment by nicola-dragoni** [comment]
Hello @helen508light, are there any updates on the situation?
I noticed I am not the only one experiencing this problem

**Comment by helen508light** [comment]
We had investigated this issue and now preparing release, which will reach prod in 2 weeks

**Comment by kenneth817** [comment]
@helen508light so will that mean 2 weeks of a broken Gantt chart visual.i am sure this will cause inconvenience to a lot of power bi users isnt it possible to perform a roll back in the interim .this chart was working perfectly yesterday.

**Comment by kenneth817** [comment]
Yes please ,kindly try for a roll back to the working version of the visual.appreciate the assistance @helen508light

**Comment by helen508light** [comment]
we had reverted this visual to working 2.0.2 version

**Comment by kenneth817** [comment]
@helen508light still shows version 2.2.2 version for me..same issue..doesn't seem to have been rectified

**Comment by helen508light** [comment]
See the powerbi web service or desktop: version have to be changed to 2.0.2
If u are still facing with 2.2.2 there - please, clear cache

**Comment by pbicvsupport** [comment]
Hi @nicola-dragoni @kenneth817 
Is this issue still reprodicable for you?
Please note, we had released new 2.2.3 version on prod
